webapp/channels/src/components/dot_menu/post_reminder_submenu.tsx

```diff
@@ -136,14 +143,18 @@ function PostReminderSubmenu(props: Props) {
                 key={`remind_post_options_${postReminder}`}
                 labels={labels}
                 trailingElements={trailingElements}
-                onClick={() => handlePostReminderMenuClick(postReminder)}
+                onClick={() => handlePostReminder(postReminder)}
+                onKeyDown={(event) => handleKeydown(event, postReminder)}
```

Comment:
Are you sure this is needed? MUI is supposed to fire the `onClick` when an item is selected by pressing space. That's how it works in the channel sidebar at least